### PR TITLE
박혜민 과제 제출

### PR DESCRIPTION
**Pull request by aimaimee**
이렇게 pull request하는게 맞을까요..? 지난주에는 잘 했는데 잘 안되네요.

-해결 못한 것-
1. train_test_split 함수를 사용하면 값이 다르게 나와서, 밑에서 매번 train, test 등의 변수를 하나하나 다시 입력했습니다.
2. y_pre_max4_mean(평균 대체)으로 한 accuracy는 맞게 나오는데 y_pre_max4_50_per(중앙값 대체)로 한 accuracy 값은 정답과 다릅니다.
3. 이상치 처리 전 값을 KDE로 그렸을 때 x축의 범위가 -250~1250으로 그려집니다.
4. 이상치 처리 후 KDE 그래프가 반대로 그려집니다.
5. y_pre_max4_mean_out 로 구한 accuracy 가 정답과 다릅니다.

### CODE REVIEW

**Comment by nuyhc** [comment]
-해결 못한 것-

1. train_test_split 함수를 사용하면 값이 다르게 나와서, 밑에서 매번 train, test 등의 변수를 하나하나 다시 입력했습니다. -> 정확한 질문과 원인이 제출한 파일에서 파악이 되지는 않습니다만, 파라미터 설정값때문에 다른거 같습니다.

2. y_pre_max4_mean(평균 대체)으로 한 accuracy는 맞게 나오는데 y_pre_max4_50_per(중앙값 대체)로 한 accuracy 값은 정답과 다릅니다.
3. 이상치 처리 전 값을 KDE로 그렸을 때 x축의 범위가 -250~1250으로 그려집니다.
4. 이상치 처리 후 KDE 그래프가 반대로 그려집니다.
5. y_pre_max4_mean_out 로 구한 accuracy 가 정답과 다릅니다.
-> 예시로 나와있는 출려과는, 데이터를 나누는 부분에서 차이가 있다면 바뀔수 있습니다
